db/people.yml

```diff
@@ -858,6 +858,33 @@ cyshi:
       group: bg
   name: Chenyang Shi
   position: scientist
+dpeters:
+  active: true
+  aka: Dom
+  avatar: https://github.com/dpeters001.png
+  bio: Dominic is a rising-senior and chemical engineering undergraduate with minors in
+  mathematics and business administration. He has experience with Linux-based and
```

Comment:
indent this block by a further two whitespaces (indentation has meaning in yaml)